src/components/views/rooms/MemberList.js

```diff
@@ -270,7 +270,7 @@ module.exports = React.createClass({
 
             // console.log("comparing " + this.memberString(memberA) + " and " + this.memberString(memberB));
 
-            if (userA.currentlyActive && userB.currentlyActive) {
+            if ((userA.currentlyActive && userB.currentlyActive) || !this._showPresence) {
```

Comment:
>// order by last active, with "active now" first.
// ...and then by power
// ...and then alphabetically.

Why don't we just order by power first instead of special casing?

Reply:
I think because the desired behaviour is that online users should still come first if presence is enabled.

Reply:
The intended behaviour of the list is to order first by online state, then by power (so offline mods don't clutter up the top of the list).  Changing that UX is an entirely separate issue and irrelevant to this PR.  This PR is just trying to fix the regression to return to the intended behaviour.

The regression here was that we were ordering by invisible(!) and invalid(!) online state if presence is disabled, which results in an incorrect ordering, hence this fix.